LunaDll/FileManager/config_manager.cpp

```diff
@@ -842,6 +842,14 @@ static void read_layout_branches(nlohmann::json &typetree,
             {
                 dst[name] = "<invalid value format>";
             }
+        }
+		else if (SDL_strncasecmp(control.c_str(), "listGroup", 9) == 0)
+		{
+			if (entry.find("children") == entry.end())
+				continue; // invalid entry: missing required key
+
+            // list-type properties don't currently support a default value. it always defaults to an empty table. maybe changed in the future
```

Comment:
Please always use spaces for indentation. Mixing tabs and spaces makes a pain. Spaces are preferable. You may want to enable displaying of whitespace in the setup of code editor.